ui/app/AppLayouts/Chat/ChatLayout.qml

```diff
@@ -12,12 +12,28 @@ StackLayout {
     id: root
 
     property RootStore rootStore
+    property var dragArea
     readonly property var contactsStore: rootStore.contactsStore
 
     property alias chatView: chatView
+
     signal importCommunityClicked()
     signal createCommunityClicked()
 
+    QtObject {
+        id: d
+
+        function displayCommunitySettingsViews() {
+            root.currentIndex = 1
+            root.dragArea.enabled = false
```

Comment:
This will break the `binding` though :)

Reply:
Yeah, this breaks the binding, and that's why the connections is kepts! :)

Reply:
That's why I'd prefer to keep just the binding and not overwrite it anywhere

Reply:
Explanation of the need of this manual assignment above!